python/tester/upload.py

```diff
@@ -51,18 +51,46 @@ def raw_path(instrument, detector, group, snap, exposure_id, filter):
 _log.setLevel(logging.DEBUG)
 
 
-def process_group(publisher, bucket, instrument, group, filter, ra, dec, kind):
-    n_snaps = INSTRUMENTS[instrument].n_snaps
+def process_group(publisher, bucket, visit_infos):
+    """Simulate the observation of a single on-sky pointing.
+
+    Parameters
+    ----------
+    publisher : `google.cloud.pubsub_v1.PublisherClient`
+        The client that posts ``next_visit`` messages.
+    bucket : `google.cloud.storage.Bucket`
+        The bucket to which to transfer the raws, once observed.
+    visit_infos : `set` [`activator.Visit`]
```

Comment:
I'm a bit nervous calling these "visit_infos", since `VisitInfo` is a different thing that doesn't align with this very well (though maybe we should make it do so?). Why not just `visits`?

Reply:
Because they're not visits either (in any of the senses of the word)! 😭 

A name like "<something>_info" feels natural to me, because these are blocks of metadata rather than identifiers (like `visit` in the Butler context). I'm willing to be flexible on the first part, though.